src/ERC712.sol

```diff
@@ -110,7 +110,7 @@ abstract contract ERC712 is IERC712 {
      * @param  signature_ The signature.
      */
     function _revertIfInvalidSignature(address signer_, bytes32 digest_, bytes memory signature_) internal view {
-        if (!SignatureChecker.isValidSignature(signer_, digest_, signature_)) revert InvalidSignature();
+        _revertIfError(SignatureChecker.validateECDSASignature(signer_, digest_, signature_));
```

Comment:
Doesn't this mean that `_revertIfInvalidSignature` is now `_revertIfInvalidECDSASignature`? And what are the implications of this change for inheriting contracts that want to revert if invalid ECDSA and ERC1271 signature?

Reply:
Indeed. I've modified the function to also check for ERC1271 signatures and return a specific error if the signature is not a valid ECDSA or ERC1271 one.
https://github.com/MZero-Labs/common/pull/20/commits/ae806801d8c690cafdd46262faa5a8cf35830307